data/scripting/infrastructure.lua

```diff
@@ -65,44 +77,45 @@ end
 -- RST
 -- .. function:: prefilled_buildings(plr, b1_descr[, b2_descr, ...])
 --
---    Create pre-filled buildings. Each description is a arrays which contain
---    building type, build field and pre-fill information. A sample usage:
+--    Create pre-filled buildings. Each description is a array which contain
+--    building type, x and y coordinates and pre-fill information.
+--    A sample usage:
 --
 --    .. code-block:: lua
 --
 --       prefilled_buildings(wl.Game().players[1],
---          {"sentry", 57, 9}, -- Sentry completely full with soldiers
---          {"sentry", 57, 9, soldier={[{0,0,0,0}]=1}}, -- Sentry with one soldier
---          {"bakery", 55, 20, inputs = {wheat=6, water=6}}, -- bakery with wares and workers
---          {"well", 52, 30}, -- a well with workers
+--          {"empire_sentry", 57, 9}, -- Sentry completely full with soldiers
+--          {"empire_sentry", 57, 9, soldier={[{0,0,0,0}]=1}}, -- Sentry with one soldier
```

Comment:
Is the key called `soldier` or `soldiers`? Also, please align the comments with each other by inserting some whitespaces.

Reply:
I don't know :-) Just changed the buildings name. But looking at places where this is used it needs to be named `soldiers`. Will change this and align the comments.

Reply:
Done